src/OrderFormAcceptanceTests.TestData/Order.cs

```diff
@@ -64,6 +65,12 @@ public Order Retrieve(string connectionString)
             return SqlExecutor.Execute<Order>(connectionString, query, this).Single();
         }
 
+        public IEnumerable<int> GetContactIdsForOrder(string connectionString)
+        {
+            var query = "(SELECT OrganisationContactId from [dbo].[Order] WHERE OrderId=@orderId) UNION (SELECT SupplierContactId from [dbo].[Order] WHERE OrderId=@orderId)";
```

Comment:
Just a few minor formatting suggestions here:

1. Capitalize `FROM` to be consistent with other SQL statements?
2. The square brackets aren't required around `dbo`.
3. Spaces around `=` operator?
4. Add a terminating semicolon.